system/Log/Handlers/ChromeLoggerHandler.php

```diff
@@ -114,8 +114,10 @@ public function __construct(array $config = [])
 		$request = Services::request(null, true);
 
 		$this->json['request_uri'] = (string)$request->uri;
-
-		Events::on('post_controller', [$this, 'sendLogs'], HOOKS_PRIORITY_HIGH);
+                
+                Events::on('post_controller', [$this, 'sendLogs'], EVENT_PRIORITY_HIGH);
+                
+		//Events::on('post_controller', [$this, 'sendLogs'], HOOKS_PRIORITY_HIGH);
```

Comment:
Yes. And I think "HOOKS_PRIORITY_HIGH" should be change to "EVENT_PRIORITY_HIGH". So I leave this comment here to let you check it.